### PR TITLE
Input validation for Position toolkit

### LINKED ISSUE #1452: Position toolkit is missing input validation
Position toolkit is currently missing input validation (checking whether latitude is \geq -90 and \leq 90, etc.)

### PR DESCRIPTION
**Pull request by gyutaepark**
Fixes #1452 .
The PR now adds a THROW_Value statement that checks whether the longitude and latitude are within acceptable range. 
Tests to confirm these exceptions were also added in the test file.

### CODE REVIEW

File: src/toolkit/position.cc [hunk 1]
```diff
@@ -25,18 +27,40 @@ double Position::longitude() const {
...
+  LonCheck(lon);
   latitude_ = SetPrecision(lat * CYCLUS_DECIMAL_SECOND_MULTIPLIER, 1);
   longitude_ = SetPrecision(lon * CYCLUS_DECIMAL_SECOND_MULTIPLIER, 1);
 }
 
+void Position::LatCheck(double lat) {
+  if (lat > 90 || lat < -90){
+    std::stringstream msg;
+    msg << "The provided latitude (" << lat
+        << ") is outside the acceptable range.";
```
Comment (scopatz) [inline src/toolkit/position.cc]: The error message should include what the acceptable range is, as well as what the value passed in was.

File: src/toolkit/position.cc [hunk 2]
```diff
@@ -25,18 +27,40 @@ double Position::longitude() const {
...
+    throw ValueError(msg.str());
+  }
+}
+
+void Position::LonCheck(double lon) {
+  if (lon > 180 || lon < -180){
+    std::stringstream msg;
+    msg << "The provided longitude (" << lon
+        << ") is outside the acceptable range.";
+    throw ValueError(msg.str());
```
Comment (scopatz) [inline src/toolkit/position.cc]: Same as above,

**Comment by scopatz** [comment]
Thanks @gyutaepark! Looks good, just the error message updates.  Also, please add a news entry for this pr: http://xon.sh/devguide.html#changelog

File: src/toolkit/position.cc (hunk 1 above)
Comment (gyutaepark) [reply src/toolkit/position.cc]: I agree. Thank you for the comment. I'll fix this right away!

File: src/toolkit/position.cc (hunk 2 above)
Comment (gyutaepark) [reply src/toolkit/position.cc]: Same as above

File: src/toolkit/position.cc (hunk 1 above)
Comment (gyutaepark) [reply src/toolkit/position.cc]: The value passed is currently included in the msg.
Thus, I've added the acceptable range using interval notation.

**Comment by gyutaepark** [comment]
@scopatz 
Thank you. I've maded the requested changes and added a news entry for the PR.
This PR is now ready for merging.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**